### ISSUE TITLE
Error when calling `Shutdown` with `fs.PathError`

### ISSUE THREAD

**Issue by gregsadetsky** [browsh-org/browsh#416]
I'm having some issues starting browsh (I'll deal with those issues separately -- they might be my fault), and the error handling is itself causing an error.

Specifically, when `browsh.xpi` is not found, [this call to Shutdown](https://github.com/browsh-org/browsh/blob/master/interfacer/src/browsh/firefox.go#L232) ends up failing at [this line](https://github.com/browsh-org/browsh/blob/master/interfacer/src/browsh/browsh.go#L96).

The specific error is:

```
panic: interface conversion: error is *fs.PathError, not *exec.Error
```

I'm too new to Go to submit a patch/suggest a solution :-) I did find [this fix](https://github.com/helm/helm/pull/1610/files) in another project that dealt with the same error, if that can be helpful. Thanks!

**Comment by tombh** [comment]
That sounds like `go.sum` (Golang's old concept of a lockfile) isn't being honoured, like maybe the wrong version of `github.com/go-errors/errors` was downloaded? Which would make sense as Golang has moved onto Go Modules and more conventionally lockfiles. Maybe a particular version of Go is needed to support `go.sum`, now that Go Modules are the default? 

Basically, the underlying problem here is that I haven't really dived into the codebase in 3 years 🫤

**Comment by gregsadetsky** [comment]
Yeah, sorry, obviously didn't mean to point out problems just for the sake of it. The code works really great -- just wanted to share in case someone else saw this and/or could open a PR.

Definitely appreciate your efforts & this was definitely not an issue that stopped me in any way. Thank you!

**Comment by tombh** [comment]
Oh, I didn't feel annoyed by you! I really welcome the feeback, because exactly as you say, it's at least searchable record and papertrail for future travellers (namely me haha). I really appreciate it, thank you 🙇‍♂️
